lib/ecmascript.ts

```diff
@@ -1861,6 +1886,43 @@ export function InterpretISODateTimeOffset(
     return GetEpochNanosecondsFor(timeZone, dt, disambiguation);
   }
 
+  // The caller wants the offset to always win ('use') OR the caller is OK
+  // with the offset winning ('prefer' or 'reject') as long as it's valid
+  // for this timezone and date/time.
+  if (offsetBehaviour === 'exact' || offsetOpt === 'use') {
```

Comment:
I forget: did we agree to standardize on British or American spellings in our code?  I have no preference, other than we should match one standard throughout if possible.

Reply:
TC39 uses British spelling in the specifications, so that's what I've been using.